third_party/terraform/resources/resource_bigquery_table.go

```diff
@@ -132,6 +128,164 @@ func bigQueryTableSchemaDiffSuppress(_, old, new string, _ *schema.ResourceData)
 	return eq
 }
 
+func bigQueryTableTypeEq(old, new interface{}) bool {
+	equivelentSet1 := []interface{}{"INTEGER", "INT64"}
+	equivelentSet2 := []interface{}{"FLOAT", "FLOAT64"}
+	equivelentSet3 := []interface{}{"BOOLEAN", "BOOL"}
+	eq0 := old == new
```

Comment:
```suggestion
    // if members are literally equivalent, return immediately.
	if old == new {
	  return true
	}
```

What do you think of having an early exit clause instead of building up `eq` values? That lowers the complexity of the function a bit, since we'd be stepping through a series of simple cases.

Reply:
I don't really agree, I think this will just make the function longer without providing additional readability. Although if you feel strongly I can swap this to a series of if elses